### PR TITLE
Get supervisor service

### PR DESCRIPTION
**Pull request by VFritsch**

### CODE REVIEW

File: setsupervisor/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Example Service Project
```
Comment (BCaspari) [inline setsupervisor/README.md:1]: Please remove this file

File: setsupervisor/disttests/main.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/*! For license information please see main.js.LICENSE.txt */
```
Comment (BCaspari) [inline setsupervisor/disttests/main.js]: do not commit js files

File: setsupervisor/disttests/main.js.map [hunk 3]
```diff
@@ -0,0 +1 @@
+{"version":3,"file":"main.js","sourceRoot":"","sources":["../main.ts"],"names":[],"mappings":";;;AAAA,iEAAwD;AAA/C,sHAAA,aAAa,OAAA;AACtB,+DAA6D;AAApD,2HAAA,mBAAmB,OAAA"}
```
Comment (BCaspari) [inline setsupervisor/disttests/main.js.map:1]: Exclude source maps

**Review by BCaspari — CHANGES_REQUESTED** [review]
Please do not commit compiled .js and .js.map files

File: setsupervisor/README.md (hunk 1 above)
Comment (VFritsch) [reply setsupervisor/README.md]: removed README.md

File: setsupervisor/disttests/main.js (hunk 2 above)
Comment (VFritsch) [reply setsupervisor/disttests/main.js]: removed committed js files

File: setsupervisor/disttests/main.js.map (hunk 3 above)
Comment (VFritsch) [reply setsupervisor/disttests/main.js.map]: removed js file

**Review by BCaspari — APPROVED** [review]

**Pull request merged**